src/test/resources/org/prebid/server/it/test-application.properties

```diff
@@ -26,6 +26,10 @@ adapters.appnexus.enabled=true
 adapters.appnexus.endpoint=http://localhost:8090/appnexus-exchange
 adapters.appnexus.pbs-enforces-gdpr=true
 adapters.appnexus.usersync.url=//usersync-url/getuid?
+adapters.applogy.enabled=true
+adapters.applogy.endpoint=http://localhost:8090/applogy-exchange
+adapters.applogy.pbs-enforces-gdpr=true
+adapters.applogy.usersync.url=//usersync-url/getuid?
```

Comment:
Although it is not used, but please use a dummy-value like other bidders - `//applogy-usersync`. Let's keep up the project style.